src/core/chart-group.ts

```diff
@@ -0,0 +1,55 @@
+import {IChartGroup, IMinimalChart} from './chart-group-types';
+
+export class ChartGroup implements IChartGroup {
+    private _charts: IMinimalChart[];
+
+    constructor () {
+        this._charts = [];
+    }
+
+    public list (): IMinimalChart[] {
+        return this._charts;
+    }
+
+    public has (chart: IMinimalChart): boolean {
+        return this._charts.indexOf(chart) >= 0;
```

Comment:
Could use [.includes()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes)

Reply:
Thanks, still getting used to all Array functions :smile:

Reply:
Changed in https://github.com/dc-js/dc.js/pull/1729/commits/395967eb7773a95d4efa6e34b959a33333fe9e39